File: ProjectForWlf_LocalDB/APITravel_In_Group/DTO/WayPoint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DTO
{
    public class WayPoint
    {
        public double LatSource { get; set; }
        public double LatDestination { get; set; }
        public double LngSource { get; set; }
        public double LngDestination { get; set; }
        public Point[] ListPoint { get; set; }


    }
}

File: project/src/app/entities/WayPoint.ts

 import {Point } from './Point';
  
export class WayPoint  {
   
    public LatSource: number;
    public LatDestination:number;
    public LngSource: number;
    public LngDestination:number;
    public ListPoint: Point[] ;
    
    constructor( LatSource: number, LatDestination:number, LngSource: number, LngDestination:number,ListPoint: Point[]  )
       { 
        this.LatSource=LatSource;
        this.LatDestination = LatDestination;
        this.LngSource=LngSource;
        this.LngDestination = LngDestination;
        this.ListPoint = ListPoint;
      
    }
 
}
